app/jenkins-for-jira-ui/src/common/analytics/analytics-client.ts

```diff
@@ -0,0 +1,145 @@
+import { view } from '@forge/bridge';
+
+const checkIfAnalyticsPackageInstalled = async (): Promise<boolean> => {
```

Comment:
This and the function below ensure that contributors can still run the app without running into any errors.